### PR TITLE
Added ping-pong example with cert-manager dependency

### PR DESCRIPTION
**Pull request by zvonkok**
We can now deploy cert-manager via helm chart aka SRO recipe, see ping-pong example as a valid CR. 
Fixed ownership of already existing resources. 
Fixed DTK issue on older releases. 
Changed the CR `driverContainer` to `build` which is more natural, since sometimes we could also build something that is not a `driver`

### CODE REVIEW

File: charts/olm/cert-manager-0.0.1/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+- files:
+  - 0001_secret.yaml
+  - 0002_issuer.yaml
+  - 0003_certificates.yaml
+  - 0004_server.yaml
+  - 0005_client.yaml
```
Comment (dagrayvid) [inline charts/olm/cert-manager-0.0.1/kustomization.yaml]: These file contents are not correct for the files in templates
```suggestion
  - 0000_namespace.yaml 
  - 0001_operatorgroup.yaml
  - 0002_subscription.yaml
  - 0003_cr.yaml 
```

File: charts/olm/cert-manager-0.0.1/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Before it can be used, the manifest/0001_secret.yaml needs to be populated with an RSA key and a certificate, which will be used as the root certificate by the Issuer. The issuer [could also be self signed](https://cert-manager.io/docs/configuration/selfsigned/).
+
+The following commands can be used to populate the secret:
+```
+openssl genrsa -out ca.key 2048
+openssl req -x509 -new -nodes -key ca.key -subj "/CN=pingpong-ca" -days 10000 -out ca.crt
+sed s"/tls.key:.*/tls.key: $$(cat ca.key|base64 -w 0)/" -i manifests/0001_secret.yaml
+sed s"/tls.crt:.*/tls.crt: $$(cat ca.crt|base64 -w 0)/" -i manifests/0001_secret.yaml
+```
```
Comment (dagrayvid) [inline charts/olm/cert-manager-0.0.1/README.md]: This file should be deleted (copy paste error I assume)

File: charts/olm/cert-manager-0.0.1/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
+name: cert-manager
+description: Simple go app to showcase cert-manager intergration
```
Comment (dagrayvid) [inline charts/olm/cert-manager-0.0.1/Chart.yaml]:
```suggestion
description: A recipe to install cert-manager via OLM
```

File: charts/olm/cert-manager-0.0.1/templates/0000_namespace.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.specialresource.metadata.name }}
```
Comment (dagrayvid) [inline charts/olm/cert-manager-0.0.1/templates/0000_namespace.yaml]:
```suggestion
  name: {{ .Values.specialresource.metadata.name }}
  
```
Fix missing newline on yaml files

File: .github/workflows/e2e.yaml [hunk 5]
```diff
@@ -56,7 +56,31 @@ jobs:
...
+                make undeploy                                                      || oc delete cm -n kube-system github.actions.lock; exit 1
+                TAG=pr-${{ steps.short-sha.outputs.sha }} make go-deploy-manifests || oc delete cm -n kube-system github.actions.lock; exit 1
+                make test-e2e                                                      || oc delete cm -n kube-system github.actions.lock; exit 1
+                make go-undeploy-manifests                                         || oc delete cm -n kube-system github.actions.lock; exit 1
+
+                oc delete cm -n kube-system github.actions.lock
+                break
+            fi
+            echo "waiting for lock"
+            sleep
```
Comment (zvonkok) [inline .github/workflows/e2e.yaml]:
```suggestion
            sleep $SECS
```

File: charts/experimental/cert-manager-1.3.0/README.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# ping-pong
```
Comment (dagrayvid) [inline charts/experimental/cert-manager-1.3.0/README.md]: This file can be deleted

File: charts/experimental/cert-manager-1.3.0/kustomization.yaml [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+- files:
+  - 0001_secret.yaml
+  - 0002_issuer.yaml
+  - 0003_certificates.yaml
+  - 0004_server.yaml
+  - 0005_client.yaml
```
Comment (dagrayvid) [inline charts/experimental/cert-manager-1.3.0/kustomization.yaml]: these contents are incorrect

File: charts/experimental/cert-manager-1.4.0/README.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+# ping-pong
```
Comment (dagrayvid) [inline charts/experimental/cert-manager-1.4.0/README.md]: This file can be deleted as well

**Pull request merged**